app/components/elements/forms/repeatable_nested_component.rb

```diff
@@ -6,7 +6,7 @@ module Forms
     # NOTE: Use the `NestedComponentPresenter` to invoke this component; do not directly instantiate it.
     class RepeatableNestedComponent < ApplicationComponent
       def initialize(form:, model_class:, field_name:, form_component:, hidden_label: false, bordered: true, # rubocop:disable Metrics/ParameterLists
-                     reorderable: false, single_field: false, fieldset_classes: [])
+                     reorderable: false, single_field: false, fieldset_classes: [], skip_tooltip: false)
```

Comment:
We now need the ability to skip tooltips in the nested components because related link tooltips in the collection and work design are in different spots.